source/menus/tools_menu.cpp

```diff
@@ -43,6 +49,33 @@ Line lines[LINE_NUM] = {
 
 };
 
+void ToolsMenu::displayTimeInfo() {
+    int hour = dKy_getdaytime_hour();
+    int min = dKy_getdaytime_minute();
+    int moonid = dKy_moon_type_chk();
+    int date = dComIfGs_getDate();
+
+    char moonphases[7][20] = {"Full",
+                              "Waning Gibbous",
+                              "Last Quarter",
+                              "Waning Crescent",
+                              "Waxing Crescent",
+                              "First Quarter",
+                              "Waxing Gibbous"};
+
+    char Time[10];
+    char Date[10];
+    char Moon[20];
+
+    tww_sprintf(Time, "%02d:%02d", hour, min);
+    tww_sprintf(Date, "date: %d", date);
+    tww_sprintf(Moon, "Moon: %d", moonphases[0]);

```

Comment:
this doesn't seem right, should be `tww_sprintf(Moon, "Moon: %s", moonphases[date % 7]);` i think?